tasks/deploy.yaml

```
@@ -4,11 +4,11 @@ variables:
     default: "amd64"
   - name: DEPLOY_IMAGE
     description: "Container image to use to run uds deploy in"
-    default: "ghcr.io/defenseunicorns/build-harness/build-harness:2.0.28"
+    default: "ghcr.io/defenseunicorns/build-harness/build-harness:2.0.34"
   - name: BUNDLE_VERSION
     description: "Version of the bundle to deploy"
     # TODO - add release-please support
-    default: "0.2.16"
+    default: "0.3.0"
```

```suggestion
    # x-release-please-start-version
    default: "0.3.0"
    # x-release-please-end
```
Add "tasks/deploy.yaml" to the extra-files entry in the release-please-config.json file